app/templates/_base_page.html

```diff
@@ -1,5 +1,8 @@
 {% extends "toolkit/layouts/_base_page.html" %}
 
+{# Import GOV.UK Frontend components for globale usage#}
+{% from "components/button/macro.njk" import govukButton %}
```

Comment:
Are we going to have to import every component we use separately?

Reply:
Yes, this pattern is used in GOV.UK Prototype Kit(https://github.com/alphagov/govuk-prototype-kit/blob/master/app/views/layout.html#L4-L30). Ideally, it would have been nice to have these imports in another file that we could just use a single import but we had difficulties with jinja recognizing the imported macros in a separate file(Happy to talk through it if you can think of a way to import it). 

Jon's spike in supplier and GOV.UK Pays implementation all import on a page by page basis. All though this is nice to see which components are used and how often, it does cause an issue for us when we migrate from govuk-frontend 2.0 and 3.0 and 3.0 has a different folder structure so we would need to do a global find and replace. You also have to know and remove which components need to import and the syntax to use. 

There is an [outstanding issue in govuk-frontend](https://github.com/alphagov/govuk-frontend/issues/1278) asking for the package to include an "all.njk" in the same way there is a way to import all css components and all javascript.

Reply:
Makes sense, thanks for the detailed explanation! I wonder why the macro import doesn't work... 🤔 Did you namespace it in the same way as the existing toolkit imports? eg 

```
{% import 'toolkit/my-macro-file.html' as foo %}
...
{% call foo.macro_name(args) %}
```

Reply:
@katstevens I think the problem is that you can't include a macro, you can only import it; ideally we would have an `all.njk` that lets you do this:

```
{% include 'components/all.njk' %}

{{ govukButton(params) }}
```

This would be very similar to what the examples are on the GOV.UK Design System. However, this doesn't work; the macros don't get included. It does work if you redefine the macros and namespace them, but then we can't use examples from the GOV.UK Design System.

Reply:
Also, we might decide to spin out a new base template that can be shared between apps that imports all macros; that's probably a future problem though.

Reply:
Ahh I see. OK, having the imports in the `_base_page` should be fine for now, anyway.